### PR TITLE
Loss reform

### PR DESCRIPTION
**Pull request by wjrbetts**

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/ct/ct600/v3/B295.scala [hunk 1]
```diff
@@ -16,12 +16,13 @@
...
+import uk.gov.hmrc.ct.ct600.v3.retriever.CT600BoxRetriever
 
 case class B295(value: Int) extends CtBoxIdentifier(name = "Total of deductions and reliefs") with CtInteger
 
-object B295 extends Linked[CP264, B295] {
-
-  override def apply(source: CP264): B295 = B295(source.value)
+object B295 extends Calculated[B295, CT600BoxRetriever] with CtTypeConverters {
+  override def calculate(boxRetriever: CT600BoxRetriever): B295 = {
+    B295(boxRetriever.b275() + boxRetriever.b285())
```
Comment (Jrmcneil) [inline src/main/scala/uk/gov/hmrc/ct/ct600/v3/B295.scala]: Does this change the expected value of B295? Also would the boxretriever in filing/frontend have to be changed to return the calculated value?

Comment (wjrbetts) [inline src/main/scala/uk/gov/hmrc/ct/ct600/v3/B295.scala]: I've put this stuff on a new branch and backed it out here. It doesn't affect the data-entry/flow half of the changes. We can work out how to release the B295 changes under the feature flag later.

File: src/main/scala/uk/gov/hmrc/ct/computations/Validators/LossesPreviousToCurrentValidation.scala [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+package uk.gov.hmrc.ct.computations.Validators
+
+import uk.gov.hmrc.ct.box.{CtTypeConverters, CtValidation, ValidatableBox}
+import uk.gov.hmrc.ct.computations.retriever.ComputationsBoxRetriever
+
+trait LossesPreviousToCurrentValidation extends CtTypeConverters {
+
+  self: ValidatableBox[ComputationsBoxRetriever] =>
```
Comment (Jrmcneil) [inline src/main/scala/uk/gov/hmrc/ct/computations/Validators/LossesPreviousToCurrentValidation.scala]: Is this trait used for anything?

Comment (wjrbetts) [inline src/main/scala/uk/gov/hmrc/ct/computations/Validators/LossesPreviousToCurrentValidation.scala]: Well spotted, no it isn't. Pushing fix

**Pull request merged**